src/lua/uri.h

```diff
@@ -5,6 +5,8 @@
  */
 #pragma once
 
+#include <stddef.h>
+
```

Comment:
We don't add anything into this header in this commit. If some .c file needs `stddef.h`, let's add the include directive into that file.